### PR TITLE
improves circuitboard docs

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

improves the documentation for circuitboards, allowing it to be picked up by visual studio hover thingy (what is that called)

## Why It's Good For The Game

docs are good, also commenting without adding a space is bad and awful dont do it

## Changelog

:cl:
/:cl:

### CODE REVIEW

File: code/game/objects/items/circuitboards/computer_circuitboards.dm [hunk 1]
```diff
@@ -195,12 +190,12 @@
 	build_path = /obj/machinery/computer/message_monitor
 
 /obj/item/circuitboard/computer/powermonitor
-	name = "Power Monitor (Computer Board)"  //name fixed 250810
+	name = "Power Monitor (Computer Board)"  // name fixed #250810
```
Comment (SuperNovaa41) [inline code/game/objects/items/circuitboards/computer_circuitboards.dm:193]: you should just remove this, it's not something that matters at all

File: code/game/objects/items/circuitboards/computer_circuitboards.dm [hunk 2]
```diff
@@ -210,7 +205,7 @@
 	build_path = /obj/machinery/computer/sat_control
 
 /obj/item/circuitboard/computer/solar_control
-	name = "Solar Control (Computer Board)"  //name fixed 250810
+	name = "Solar Control (Computer Board)"  // name fixed #250810
```
Comment (SuperNovaa41) [inline code/game/objects/items/circuitboards/computer_circuitboards.dm:208]: same here

File: code/game/objects/items/circuitboards/computer_circuitboards.dm [hunk 3]
```diff
@@ -241,7 +236,7 @@
 	greyscale_colors = CIRCUIT_COLOR_GENERIC
 	build_path = /obj/machinery/computer/arcade/orion_trail
 
-/obj/item/circuitboard/computer/holodeck// Not going to let people get this, but it's just here for future
+/obj/item/circuitboard/computer/holodeck // Not going to let people get this, but it's just here for future
```
Comment (SuperNovaa41) [inline code/game/objects/items/circuitboards/computer_circuitboards.dm:239]: probably isn't needed either, a lot of these are just like coder thoughts but not actually needed for anyone to work with the code (aka REMOVE THEM)

**Comment by JohnFulpWillard** [comment]
changelog meant for player-facing stuff 😦

**Pull request closed without merging**